src/components/Card.js

```diff
@@ -135,6 +136,11 @@ const Card = ({ answer, term, explanation, resource, page, search, index, listLe
   // eslint-disable-next-line
   let title;
 
+  // For Cards with a CardLink, that element will get focus during keyboard
+  // navigation. When that happens, we'll update the Card component's state so
+  // we can give the entire Card a focus style.
+  const [focused, setFocused] = useState(false);
+
```

Comment:
Another potential way of fixing this is by using refs. If you look at lines 89, 122-126, 140, 143-147 in `src/components/SearchBar.js`, you'll see it done that way with the conditional on the 'clear search' button.

That said, both ways work just fine and I'm fine with keeping as is.

Reply:
I wasn't able to get this working because the component wasn't re-rendering on link focus...it makes sense for the input because stuff happens on change but maybe it won't work in this instance? Let me know if I'm missing something. Also, can I just say how much I appreciate all the line numbers ✨

Reply:
Ah, interesting. Well like I said, good with keeping it as is!